packages/eslint-plugin/src/configs/all.json

```diff
@@ -0,0 +1,57 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "sourceType": "module"
+  },
+  "plugins": ["@typescript-eslint"],
```

Comment:
what you do you think about creating `base.json` and extends other configs over it.

base config should set parser, options, plugins and turn off not compatible eslint rules.